README.md

```diff
@@ -1,148 +1,109 @@
 
+<br/>
 
+Vizia is a (in development) declarative GUI framework for the Rust programming language.
 
-VIZIA is a (in development) declarative GUI framework for the Rust programming language.
+> **WARNING** - Vizia is currently experimental and *not* considered stable.
 
-**WARNING** - VIZIA is currently experimental and *not* a fully functioning framework. The code in this repositiory is not considered stable and is likely to change.
-
-## Views
-Views form the basic building blocks of a GUI. A view could describe a widget like a label or button, or a more complex container such as a list.
-
-### Composing Views
-Views can be easily composed to form a more complex GUI application:
 ```rust
-Application::new(WindowDescription::new(), |cx|{
-	HStack::new(cx, |cx|{
-		Label::new(cx, "Hello");
-		Label::new(cx, "World");
-	});
-}).run();
-```
+#[derive(Lens)]
+pub struct AppData {
+    count: i32,
+}
 
-### Layout and Styling
-Inline properties used for layout and styling can be set directly on views:
-```rust
-Application::new(WindowDescription::new(), |cx|{
-	HStack::new(cx, |cx|{
-		Label::new(cx, "Hello")
-			.width(Pixels(200.0));
-		Label::new(cx, "World")
-			.background_color(Color::blue());
-	}).col_between(Pixels(10.0));
-}).run();
-```
-Styling can also be done with CSS stylesheets:
-```css
-/* style.css */
-label {
-	width: 100px;
-	height: 30px;
-	border-width: 1px;
-	border-color: black;
+pub enum AppEvent {
+    Increment,
 }
 
-.foo {
-	background-color: red;
+impl Model for AppData {
+    fn event(&mut self, _: &mut Context, event: &mut Event) {
+        if let Some(app_event) = event.message.downcast() {
+            match app_event {
+                AppEvent::Increment => {
+                    self.count += 1;
+                }
+            }
+        }
+    }
 }
-```
-```rust
-Application::new(window_description, |cx|{
 
-	cx.add_style("style.css");
+Application::new(WindowDescription::new(), |cx| {
+    AppData { count: 0 }.build(cx);
 
-	HStack::new(cx, |cx|{
-		Label::new(cx, "Hello");
-		Label::new(cx, "World").class("foo");
-	});
+    HStack::new(cx, |cx| {
+        Button::new(cx, |cx| cx.emit(AppEvent::Increment), |cx| Label::new(cx, "Increment"));
+        Label::new(cx, AppData::count).width(Pixels(50.0));
+    })
+    .child_space(Stretch(1.0))
+    .col_between(Pixels(50.0));
 }).run();
 ```
-## State
-State describes the data provided by the user which the GUI will modify. VIZIA is reactive, which means that changes to the state will update the views.
- 
-### Defining State
-State is defined as a struct/enum which implements the `Model` trait:
-```rust
-#[derive(Lens)]
-struct AppData {
-	count: i32,
-}
 
-impl Model for AppData {}
+## Features
+ - Multiplatform (Windows, Linux, MacOS, Web)
+ - Declarative API
+ - Reactive with an elm-like archtecture
+ - Flexible layout engine, powered by [morphorm](https://github.com/vizia/morphorm)
+ - CSS-like styling with hot reloading
```

Comment:
We're not css-like, we're actually using css! I don't think that just because we don't have the same properties as html we have to downplay it here.

Reply:
If you want to make it clear that we don't support the same style properties as html I would say "HTML-like CSS styling"

Reply:
Okay good point. Though it's not just the difference in properties, there's quite a few selector things we don't support (yet) as well.